Oloughlin_Transmissivity_Calculation

```diff
@@ -404,7 +391,7 @@ def final_graph(T, df, b, a1):
     df2['logTI'] = np.log(df2['TI'])
     plt.scatter(df2['logT/Qo'],df2['0-1 Ol'],c=df2.index, s=50, cmap='Blues', alpha = 1)
```

Comment:
I don't know about these labels like '0-1 01' - could they be more descriptive? This just means event runoff ratio, right? it could be 'RR_event' or something similar